opentelemetry-sdk/src/testing/trace/span_exporters.rs

```diff
@@ -40,42 +42,47 @@ pub fn new_test_export_span_data() -> SpanData {
     }
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
 pub struct TestSpanExporter {
-    tx_export: Sender<SpanData>,
-    tx_shutdown: Sender<()>,
+    pub export_called: Arc<Mutex<bool>>,
+    pub shutdown_called: Arc<Mutex<bool>>,
```

Comment:
We can use `AtomicBool` for simpliciy

Reply:
https://github.com/open-telemetry/opentelemetry-rust/pull/1612#discussion_r1520864465